x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/support/SecurityIndexManager.java

```diff
@@ -63,42 +62,68 @@
 import static org.elasticsearch.xpack.core.ClientHelper.executeAsyncWithOrigin;
 
 /**
- * Manages the lifecycle of a single index, its template, mapping and and data upgrades/migrations.
+ * Manages the lifecycle of a single index, its mapping and and data upgrades/migrations.
  */
 public class SecurityIndexManager implements ClusterStateListener {
 
-    public static final String INTERNAL_SECURITY_INDEX = ".security-" + IndexUpgradeCheckVersion.UPRADE_VERSION;
-    public static final int INTERNAL_INDEX_FORMAT = 6;
+    public static final String SECURITY_ALIAS_NAME = ".security";
+    public static final int INTERNAL_SECURITY_INDEX_FORMAT = 6;
```

Comment:
shouldn't we increment the version post upgrade?

Reply:
Yes, this is done in `SecurityIndexUpgradeCheck#upgrade` . Again, this is the face of `SecurityIndexManager` for 6.7 . In the 7 backport the number here will change to 7.